settings/Case.setting.php

```diff
@@ -100,4 +100,19 @@
     'description' => 'How to sort activity-types on the "Manage Case" screen? (Set "Default" to load setting from the legacy "Settings.xml" file.)',
     'help_text' => '',
   ),
+  'civicaseActivityRevisions' => array(
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'civicaseActivityRevisions',
+    'type' => 'Boolean',
+    'quick_form_type' => 'YesNo',
+    'default' => TRUE,
```

Comment:
Trade-off:
 * For existing CiviCase users, `TRUE` maintains the status-quo. It's arguably buggy, but it's the bug they know -- and sometimes folks have implicit dependencies on a long-standing quirks.
 * For new CiviCase adopters, `FALSE` will leave their data ready to go to CiviCase 5 without any extra migration steps. However, if we're going to change the default, then we really need a more active round of communications.